client.go

```diff
@@ -1,101 +1,118 @@
 package gremtune
 
 import (
+	"fmt"
 	"io/ioutil"
 	"log"
 	"sync"
 	"time"
 
 	"github.com/pkg/errors"
+	"github.com/schwartzmx/gremtune/interfaces"
```

Comment:
Shouldn't this refer our url?

Reply:
No. As discussed it has to refer to the repo stated in the go.mod file.
The whole renaming to go-gremlin-cosmos will be done with #5